sigs.yml

```diff
@@ -350,7 +350,7 @@
     - name: Trask Stalnaker
       github: trask
   - name: Collector
-    meeting: "Wednesdays, rotating between the following time slots: 00:00 UTC, 12:00 UTC, 16:00 UTC"
+    meeting: Wednesday alternating between 09:00 PT, 17:00 PT, 20:00 UTC+8
```

Comment:
@open-telemetry/collector-maintainers please take a look if I did the time conversion correctly